jira_agile_metrics/calculators/progressreport.py

```diff
@@ -443,11 +437,14 @@ def calculate_team_throughput(
     frequency
 ):
 
+    backlog_column_index = [s['name'] for s in cycle].index(backlog_column)
+    committed_column = cycle[backlog_column_index + 1]['name']  # config parser ensures that `backlog` comes immediately before `committed`
```

Comment:
This looks weird. Why did you make this change? Don't have a configuration parameter to identify the committed column? And also, I'm not sure the config parser does (intentionally?) ensure it's always "backlog column+1", that's just the default if the committed-column isn't specified.

Reply:
See https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#discussion_r432833831 the implicit assumption was the following:
```
started_column = cycle[backlog_column_index + 1]['name']  # config parser ensures there is at least one column after backlog
```
I made this explicit in `config.py` ensuring that `backlog_column` is immediately preceding `committed_column`. These implicit assumptions confused me a lot when refactoring the code.